package.json

```diff
@@ -26,25 +20,26 @@
     "@types/qrcode.react": "^1.0.1",
     "amplitude-js": "^7.3.0",
     "bignumber.js": "^9.0.1",
-    "lodash": "^4.17.20",
+    "lodash": "^4.17.21",
     "moment": "^2.29.0",
     "qrcode.react": "^1.0.0",
-    "react": "^17.0.1",
+    "react": "^17.0.2",
     "react-copy-to-clipboard": "^5.0.2",
-    "react-dom": "^17.0.1",
-    "react-redux": "^7.2.0",
+    "react-dom": "^17.0.2",
+    "react-redux": "^7.2.3",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.2",
+    "react-scripts": "4.0.3",
     "redux": "^4.0.5",
     "stellar-identicon-js": "^1.0.0",
     "stellar-sdk": "^6.0.0",
-    "styled-components": "^5.2.0",
-    "trezor-connect": "^8.1.16",
-    "typescript": "~4.1.3"
+    "styled-components": "^5.2.3",
+    "trezor-connect": "^8.1.26",
+    "tslib": "^2.2.0",
```

Comment:
Added `tslib` directly to solve [this issue](https://github.com/microsoft/TypeScript/issues/37991).